examples/cpp/CPUDistribution.cc

```diff
@@ -14,17 +14,27 @@
 #include <cstdlib>
 #include <iomanip>
 #include <iostream>
+#include <memory>
 #include <string>
 
 #include "BPF.h"
+#include "shared_table.h"
+#include "table_desc.h"
+#include "common.h"
+
+const std::string BPF_TABLES = R"(
```

Comment:
Umm, that was me playing around making sure this worked with the C++ API. I intend to remove it.